fuse.py

```diff
@@ -55,13 +72,29 @@ class c_utimbuf(Structure):
 class c_stat(Structure):
     pass    # Platform dependent
 
-_system = system()
-_machine = machine()
-
 if _system == 'Darwin':
     _libiconv = CDLL(find_library('iconv'), RTLD_GLOBAL) # libfuse dependency
     _libfuse_path = (find_library('fuse4x') or find_library('osxfuse') or
                      find_library('fuse'))
+elif _system == 'Windows':
```

Comment:
can we put this in a different PR? As of #31 there is the ENV var `FUSE_LIBRARY_PATH`

Reply:
I have looked at the FUSE_LIBRARY_PATH changes. Perhaps we could handle this similarly to how Darwin is handled? See below:

```python
_libfuse_path = os.environ.get('FUSE_LIBRARY_PATH')
if not _libfuse_path:
    if _system == 'Darwin':
        # Darwin code
    elif _system == 'Windows':
        # Windows code
    else:
        _libfuse_path = find_library('fuse')
```

Unfortunately when I created this PR I created it off the `master` branch in my fork and there are now users that rely on that branch. This makes it hard to split the PR into two (at least with my limited GitHub skills). It might be easier to put everything in the same PR (unless you do not intend to incorporate this particular change into the main project).

Reply:
@billziss-gh I was mainly interested in merging in the changes sans registry code and wanted to start a separate discussion on the registry related changes.

If you're not too familiar with git I can probably look at squashing your changes and creating a new PR where @eestrada and others can discuss the change.

Reply:
Understood. I will create a new branch with just the Windows/Cygwin `stat` related changes (and without the registry code) and probably end up submitting as a new PR. I am currently very busy with multiple projects, but I am hoping to have this for you this or next week.

Reply:
@billziss-gh thank you! :slightly_smiling_face: